RestaurantsNearMe/Presentation/RestaurantListViewModel/RestaurantListRow.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  RestaurantListRow.swift
+//  RestaurantsNearMe
+//
+//  Created by Colin Evans on 2024-01-24.
+//
+
+import SwiftUI
+
+struct RestaurantListRow: View {
+  let restaurant: Restaurant
+  @Environment(\.displayScale) var scale
+
+  var body: some View {
+    GeometryReader { proxy in
+      HStack(spacing: proxy.size.width * UIConstants.widthRatio){
+        AsyncImage(
```

Comment:
My understanding of this type is the underlying View handles loading, cancelling and caching of the image. Which means no additional networking logic should be required here.

Does that match most uses cases?